### PR TITLE
Notify about open reconciliation PRs for prerelease version as well

### PR DESCRIPTION
**Pull request by thegreyd**

### CODE REVIEW

File: jobs/build/nag-upstream/Jenkinsfile [hunk 1]
```diff
@@ -37,44 +37,40 @@ node() {
...
-                    } else {
-                        echo "There are no alignment prs left open."
-                        if (params.SEND_TO_SLACK) {
-                            releaseChannel.say(":scroll: All prs are merged for ${group}")
+    def include_master = software_lifecycle_phase == "pre-release" ? '--include-master' : ''
+    timestamps {
+        releaseChannel = slacklib.to(BUILD_VERSION)
+        try {
+            withCredentials([string(credentialsId: 'openshift-bot-token', variable: 'GITHUB_TOKEN')]) {
+                report = buildlib.doozer("${doozerOpts} images:streams prs list ${include_master}", [capture: true]).trim()
```
Comment (thegreyd) [inline jobs/build/nag-upstream/Jenkinsfile:45]: relevant line that changed in block, except this everything is identical

File: jobs/build/nag-upstream/Jenkinsfile [hunk 2]
```diff
@@ -37,44 +37,40 @@ node() {
     // Skip the build for versions in eol/pre-release state
     def out = buildlib.doozer("${doozerOpts} config:read-group --yaml software_lifecycle", [capture: true]).trim()
     def software_lifecycle_phase = readYaml(text: out)["phase"]
-
-    if (software_lifecycle_phase != 'release') {
-        currentBuild.displayName += " [skipped]"
-    } else {
```
Comment (thegreyd) [inline jobs/build/nag-upstream/Jenkinsfile:43]: removed this

File: jobs/build/nag-upstream/Jenkinsfile [hunk 3]
```diff
@@ -37,44 +37,40 @@ node() {
...
-                        ]
-                        releaseChannel.pinAttachment(attachment)
-                        if (params.SEND_TO_SLACK) {
-                            releaseChannel.say(":scroll: Howdy! Some alignment prs are still open for ${group}\n", [attachment])
-                        }
-                    } else {
-                        echo "There are no alignment prs left open."
-                        if (params.SEND_TO_SLACK) {
-                            releaseChannel.say(":scroll: All prs are merged for ${group}")
+    def include_master = software_lifecycle_phase == "pre-release" ? '--include-master' : ''
```
Comment (joepvd) [inline jobs/build/nag-upstream/Jenkinsfile:40]: maybe return if software_lifecyle is not in `[release, pre-release]`. 

Maybe we'll grow an `eol` here, or something else.

Comment (thegreyd) [reply jobs/build/nag-upstream/Jenkinsfile:40]: Makes sense :+1:

**Comment by thegreyd** [comment]
received a verbal lgtm from joep
merging

**Pull request merged**